server/src/main/java/org/apache/kafka/server/config/KafkaConfig.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.server.config;
+
+import org.apache.kafka.common.config.TopicConfig;
+
+public class KafkaConfig {
+    private final static String LOG_CONFIG_PREFIX = "log.";
```

Comment:
Minor 

1. Let's rename this to `LOG_PREFIX` to make naming more compact.

2. We have `ServerTopicConfigSynonyms.LOG_PREFIX`, already.
Maybe it will be better to use only one constant?

Reply:
Nice catch. I updated the pr to `ServerTopicConfigSynonyms.LOG_PREFIX`